lte/gateway/python/scripts/BUILD.bazel

```diff
@@ -95,6 +95,15 @@ py_binary(
     ],
 )
 
+# Note: Not included in the magma debian package. Only used for integration tests.
+py_binary(
+    name = "config_iface_for_ipv6",
+    srcs = ["config_iface_for_ipv6.py"],
+    imports = [ORC8R_ROOT],
+    legacy_create_init = False,
+    deps = ["//orc8r/gateway/python/magma/configuration:service_configs"],
+)
```

Comment:
This is the main part of the fix?

Reply:
Yes, now the script is build with bazel and can be used by the integration tests.